data/data-pipeline/data_pipeline/etl/sources/tree_equity_score/etl.py

```diff
@@ -7,6 +7,19 @@
 
 
 class TreeEquityScoreETL(ExtractTransformLoad):
+    """
```

Comment:
love these docstrings! 

(if you want, same comment as rounding PR: move a <= 1-line summary of this class to the first line.)